src/main/java/frc/robot/robotcontainers/NewRobotContainer.java

```diff
@@ -88,7 +93,8 @@ public NewRobotContainer() {
         )
     );
 
-    intake.setDefaultCommand(intake.autoIntake());
+    intake.setDefaultCommand(intake.defaultIntakeCommand());
```

Comment:
Add a boolean flag to switch between 1) simple manual intake/shooter 2) state based intake/shooter.